styles.css

```diff
@@ -374,6 +374,96 @@
   .contact-inner-label-desktop {
     display: none;
   }
+
+  .elementpop {
+    display: flex;
+    flex-direction: column;
+    background-color: #fff;
+    height: 100%;
+    width: 100%;
+    border: 2px solid gray;
+  }
+
+  .popup-body {
+    display: flex;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 9999;
+    justify-content: center;
+    align-content: center;
+    background-color: rgba(0, 0, 0, 0.5);
+    backdrop-filter: 8px;
+    opacity: 0;
+    visibility: hidden;
+  }
+
+  .popup-img {
+    height: 30%;
+    margin: auto;
+  }
+
+  .showpopup {
+    opacity: 1;
+    visibility: visible;
+  }
+
+  .pop-para {
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    padding-left: 12px;
+  }
+
+  .content-inner-pop {
+    display: flex;
+    flex-direction: row;
+    padding-inline-start: 0;
+  }
+
+  .pop-btn {
+    margin: 5px;
+    background-color: #e7e7e7;
+    border-radius: 5px;
+  }
```

Comment:
- [ ] Hi to improve the UI ill recommend giving your `pop-btn` padding to centralize the text.